integration_tests/commands/hstrlen_test.go

```diff
@@ -0,0 +1,49 @@
+package commands
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestHSTRLEN(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+	defer FireCommand(conn, "DEL key_hStrLen key")
+
+	testCases := []TestCase{
+		{
+			commands: []string{"HSTRLEN", "HSTRLEN KEY", "HSTRLEN KEY FIELD ANOTHER_FIELD"},
+			expected: []interface{}{"ERR wrong number of arguments for 'hstrlen' command",
+				"ERR wrong number of arguments for 'hstrlen' command",
+				"ERR wrong number of arguments for 'hstrlen' command"},
+		},
+		{
+			commands: []string{"HSET key_hStrLen field value", "HSET key_hStrLen field HelloWorld"},
+			expected: []interface{}{ONE, ZERO},
+		},
+		{
+			commands: []string{"HSTRLEN wrong_key_hStrLen field"},
+			expected: []interface{}{ZERO},
+		},
+		{
+			commands: []string{"HSTRLEN key_hStrLen wrong_field"},
+			expected: []interface{}{ZERO},
+		},
+		{
+			commands: []string{"HSTRLEN key_hStrLen field"},
+			expected: []interface{}{int64(10)},
+		},
+		{
+			commands: []string{"SET key value", "HSTRLEN key field"},
+			expected: []interface{}{"OK", "WRONGTYPE Operation against a key holding the wrong kind of value"},
+		},
```

Comment:
The test cases here depend on each other (the fifth test case will not pass unless the second case is run). Each test case should ideally be hermetic, so that it can be run individually.

Reply:
Fixed the test cases and rebased. Thanks @JyotinderSingh.